vm/src/obj/objint.rs

```diff
@@ -124,7 +124,7 @@ fn inner_pow(int1: &BigInt, int2: &BigInt, vm: &VirtualMachine) -> PyResult {
         objfloat::float_pow(v1, v2, vm).into_pyobject(vm)
     } else {
         Ok(if let Some(v2) = int2.to_u64() {
-            vm.ctx.new_int(int1.pow(v2))
+            vm.ctx.new_int(Pow::pow(int1, v2))
```

Comment:
is this related to bignum 0.3?

Reply:
Yeah, it adds a pow method directly on BigInt that takes a u32; I'm not sure why.